compiler/circle2circle/src/Circle2Circle.cpp

```diff
@@ -247,6 +247,11 @@ int entry(int argc, char **argv)
     .help("This will convert weight format of FullyConnected to SHUFFLED16x1FLOAT32. Note that "
           "it only converts weights whose row is a multiple of 16");
 
+  arser.add_argument("--replace_non_const_fc_with_batch_matmul")
```

Comment:
Q) which would be correct? `fc_with_batch_matmul` or `fc_with_batchmatmul` ?

Reply:
In comments it is usually spelled with a space

Reply:
```suggestion
  arser.add_argument("--replace_non_const_fc_with_batchmatmul")
```